addons/languages/piedmontese/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```

Comment:
Have this as false for now.

Reply:
@glemco please also set this to `false` for now. After reviewing we (the maintainers) we'll set this to `true`